cogs/bot.py

```diff
@@ -162,11 +162,11 @@ async def on_command_error(self, ctx, error):
                 value="If, after reading and understanding the reason provided above, you believe your account was suspended in error, and that you did not violate the Terms of Service, you may submit a [Bot Suspension Appeal](https://forms.poketwo.net/a/suspension-appeal) to request a re-review of your case.",
                 inline=False,
             )
-            await ctx.send(embed=embed)
+            await ctx.reply(embed=embed)
         elif isinstance(error, checks.AcceptTermsOfService):
             ctx.log.info("command.error.AcceptTermsOfService")
         elif isinstance(error, (commands.CheckFailure, commands.UserInputError, flags.ArgumentParsingError)):
-            await ctx.send(error)
+            await ctx.reply(error)
```

Comment:
Could you do `mention_author=False` for this because we don't want users' mentions tabs to get clogged.